service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperControllerSubmittedTest.java

```diff
@@ -64,31 +65,32 @@ void shouldNotifyMultipleGatekeepersWithExpectedNotificationParameters() throws
 
         verify(notificationClient).sendEmail(
             GATEKEEPER_SUBMISSION_TEMPLATE, GATEKEEPER_EMAIL,
-            buildExpectedParameters(CAFCASS_EMAIL), "12345");
+            getTemplate(CAFCASS_EMAIL).toMap(mapper), "12345");
 
         verify(notificationClient).sendEmail(
             GATEKEEPER_SUBMISSION_TEMPLATE, CAFCASS_EMAIL,
-            buildExpectedParameters(GATEKEEPER_EMAIL), "12345");
+            getTemplate(GATEKEEPER_EMAIL).toMap(mapper), "12345");
```

Comment:
Why not just verify against the NotifyTemplate version of the method call? You can capture the data in an ArgumentCaptor and compare against it there. I did something similar in the handler test. Although thinking about it, does this test need to worry about the content of the email sent, could you just verify that an email was sent twice?

Reply:
Potentially no, we test the content in the handler itself. I'm happy to remove the assertion around content for the int test.